sdk/src/test/java/com/qonversion/android/sdk/internal/QonversionInternalTest.kt

```diff
@@ -255,4 +263,94 @@ internal class QonversionInternalTest {
             }
         }
     }
+
+    @Nested
+    inner class GetUserInfoTest {
+        private val mockUserController = mockk<UserController>()
+
+        @BeforeEach
+        fun setUp() {
+            every {
+                mockDependenciesAssembly.userController()
+            } returns mockUserController
+        }
+
+        @ExperimentalCoroutinesApi
```

Comment:
This can be added for the whole class (either inner or global), so there won't be any warning in other tests.